bonds.md

```diff
@@ -1,17 +0,0 @@
-# Bonds
-
-## What are bonds?
-
-**Bonds are our secondary strategy. They provide a more conservative and reliable return.** Bonds are a cross between a fixed income product, a futures contract, and an option. 
-
-The protocol quotes the bonder with terms for a trade at a future date. These terms include a defined amount of OHM the bonder will receive and the time when vesting is complete. The bond becomes redeemable as it vests. I.e. 2 days in 40% of the reward can be claimed.
-
-**This allows Olympus to accumulate its own liquidity.** Doing so ensure there's always locked liquidity in the pool to facilitate market operations and protect token holders.
-
-\*\*\*\*
-
-\*\*\*\*
-
-\*\*\*\*
-
-\*\*\*\*
```

Comment:
Dropping this. Never knew what this was good for. Looked super broken all the time.